src/d3a/models/appliance/inter_area.py

```diff
@@ -52,6 +55,8 @@ def event_tick(self):
         )
 
     def slot_energy(self, market):
+        if not hasattr(market, "trades"):
```

Comment:
This was really needed at the end ?

Reply:
Ah no, didnt try it. I thought we decided to keep it anyway. Shall I remove it?

Reply:
I am not sure if it is even worth your time. If you can quickly try it out then give it a go, otherwise leave it, I am fine with either.

Reply:
I just tried it again. We indeed need it.

Reply:
Thanks for checking it out!